### PR TITLE
[feat: basic logging integration

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: sentry_sdk/integrations/logging.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+        event['level'] = record.levelname.lower()
+        event['logger'] = record.name
+
+        event['logentry'] = {
+            'message': to_string(record.msg),
+            'params': record.args
+        }
+
+        # TODO: also send formatted message
```
Comment (untitaker) [inline sentry_sdk/integrations/logging.py]: Unclear where this should go. @jan-auer said `event['message']` was deprecated, but it's used in this sdk already. Let's talk tomorrow.

Comment (mitsuhiko) [reply sentry_sdk/integrations/logging.py]: message is not deprecated but you can use logentry if you have messages with parameters (only %s is supported though).

Comment (untitaker) [reply sentry_sdk/integrations/logging.py]: I already do that. I was under the impression that the integration is supposed to send both the formatted and unformatted message. Is this wrong?

Comment (mitsuhiko) [reply sentry_sdk/integrations/logging.py]: It only needs to send one or the other.

**Pull request merged**